tests/pytests/unit/pillar/test_pillar.py

```diff
@@ -173,5 +173,6 @@ def test_remote_pillar_timeout(temp_salt_minion, tmp_path):
     mock = MagicMock()
     mock.side_effect = salt.exceptions.SaltReqTimeoutError()
     pillar.channel.crypted_transfer_decode_dictentry = mock
+    msg = r"^Pillar timed out after \d{1,4} seconds$"
```

Comment:
This never gets used

Reply:
This is a merge forward, the fix needs to happen on older branches getting merged forward.